src/server/api/routers/users.ts

```diff
@@ -40,12 +35,18 @@ export const userRouter = createTRPCRouter({
 			}),
 		)
 		.mutation(async ({ ctx, input }) => {
+			const userId = ctx.session.user.id;
+			const user = await ctx.prisma.user.findUnique({
+				where: {
+					id: userId,
+				},
+			});
```

Comment:
Same as above. The build is failing because you need to check that a user was returned before you access the `role` property.
```js
if (!user) {
	throw new Error("User not found");
}
```